frontend/apps/reader/modules/readerannotation.lua

```diff
@@ -254,6 +254,10 @@ function ReaderAnnotation:updatePageNumbers(force_update)
     self.needs_update = nil
 end
 
+function ReaderAnnotation:onUseReferencePageUpdate()
```

Comment:
I think it's enough:
`ReaderAnnotation.onUseReferencePageUpdate = ReaderAnnotation.setNeedsUpdateFlag`